edi_oca/static/src/js/widget_edi.esm.js

```diff
@@ -23,8 +24,14 @@ export class EdiConfigurationWidget extends Component {
         );
         this.action.doAction(action);
     }
+
+    getInfo() {
```

Comment:
```suggestion
    get info() {
```

Reply:
if you want to have a getInfo, maybe it is better to have a get funcion, isn't it :wink: